forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -213,9 +215,10 @@ open class DeviceBindingCallback : AbstractCallback, Binding {
                                  deviceBindingRepository: DeviceBindingRepository = LocalDeviceBindingRepository(
                                      context),
                                  deviceId: String = DeviceIdentifier.builder().context(context)
-                                     .build().identifier) {
+                                     .build().identifier,
+                                 prompt: Prompt? = null) {
```

Comment:
Update method documentation

Reply:
@vahancouver i think the test chages are not committed accidentally. pushed it now